modules/lymphgen/1.0/lymphgen.smk

```diff
@@ -64,11 +89,46 @@ def best_seg(mod_config):
         
         mod_config["runs"].at[index, "cnv_path"] = df[index]["cnv_path"]
 
+    callers_sv = list(mod_config["inputs"]["sample_sv_info"]["other"].keys())
+    callers_sv = [caller.lower() for caller in callers_sv]
+    dfm = mod_config["runs"]
+    df = dfm.to_dict('index')
+    for index in df.keys():
+        possible_outputs = []
+        for caller in callers_sv:
+            if caller == "empty_sv":
+                possible_output = [mod_config["inputs"]["sample_sv_info"]["other"][caller]]
+            else:
+                possible_output = (expand(mod_config["inputs"]["sample_sv_info"]["other"][caller], zip,
+                                        tumour_id=df[index]["tumour_sample_id"],
+                                        normal_id=df[index]["normal_sample_id"],
+                                        pair_status=df[index]["pair_status"],
+                                        genome_build=df[index]["tumour_genome_build"],
+                                        seq_type=df[index]["tumour_seq_type"]))
+
+            if os.path.exists(possible_output[0]):
+                possible_outputs += possible_output
+
+        df[index]["sv_path"] = possible_outputs[0]
+
+        mod_config["runs"].at[index, "sv_path"] = df[index]["sv_path"]
+
     all = mod_config["runs"]
+
+    all['has_sv'] = False
+    for i, row in all.iterrows():
+        if (row['tumour_seq_type'] == "capture") & (row['sv_path'] != config["lcr-modules"]["lymphgen"]["inputs"]["sample_sv_info"]["other"]["empty_sv"]):
```

Comment:
Just confirming the logic here because I'm not sure I understand this: 

- If a sample is capture, it should only go through the `with_sv` pipeline if it has FISH
- If a sample is genome, it should go through `with_sv` whether it has FISH OR svar_master OR manta outputs

Is this how it's working?

Reply:
Could you add a few comments to keep track of that logic so future users/editors know what's going on and why?